int/api/myplugin/execute.go

```diff
@@ -25,35 +24,37 @@ func (e *execute) Handle(params operations.PluginManagerExecuteCommandParams) mi
 
 	config.Logger.Debugf("[POST /plugin-manager/%s/execute] command: %s", params.ID, cmd)
 
-	plugin, err := e.manager.Plugin(params.ID)
+	plgn, err := e.manager.Plugin(params.ID)
```

Comment:
I don't really like removing letters from a word in variable names
I think you should rename it to `plugin` and import the `plugin` package as `pluginPkg` for example (in the case of the import, it's fine to remove letter from "package" because the directory is already named `pkg`)

```suggestion
	plugin, err := e.manager.Plugin(params.ID)
```

Reply:
done